shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -921,6 +927,42 @@ - (void)waitForFirstFrame:(NSTimeInterval)timeout
   });
 }
 
+- (FlutterEngine*)spawnWithEntrypoint:(NSString*)entrypoint {
+  assert(_shell);
```

Comment:
then the doc should be clear about when this call should be allowed, even if it's private.

Reply:
Also, be consistent within the file (which seems to use NSAsserts everywhere)

Reply:
addressed in "xiao feedback 1"